tools/project_manager/diff.php

```diff
@@ -138,11 +141,26 @@
     echo "| <a href='$format_url'>$format_label</a>\n";
 }
 
+if (user_can_mentor_in_any_round()) {
+    // show option to change diff style
+    if ($bb_diffs) {
+        $bb_diffs_label = _("Table diff style");
```

Comment:
Should we make this text more action-centric since they're links?
`Show table-style diff` and `Show BBCode-style diff`